demes/renv/activate.R

```diff
@@ -0,0 +1,994 @@
+
```

Comment:
This is a big file that looks like it might be autogenerated. Or unrelated? Is this `renv` folder needed?

Reply:
I did some reading and renv is there to keep track of dependency versions and the like. On their [website](https://rstudio.github.io/renv/articles/renv.html) it states:

> For development and collaboration, the .Rprofile, renv.lock and renv/activate.R files should be committed to your version control system; the renv/library directory should normally be ignored. Note that [renv::init()](https://rstudio.github.io/renv/reference/init.html) will attempt to write the requisite ignore statements to the project .gitignore.

So, I think it is supposed to be there. slendr does not have a renv folder though. @bodkan Was this a conscious decision and do you have any other thoughts on the matter?